packages/components/src/components/forms/ImageUploader.tsx

```diff
@@ -46,15 +46,19 @@ export class ImageUploader extends React.Component<IImagePickerProps, {}> {
   fileUploader: React.RefObject<HTMLInputElement> = React.createRef()
   handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     const { files } = event.target
-    return files && this.props.handleFileChange(files[0])
+    if (files) {
+      this.props.handleFileChange(files[0])
+    }
+    // Required to select the same file again
+    event.target.value = ''
```

Comment:
Previously after deleting an uploaded file, it was not possible to upload that file again.